README.md

```diff
@@ -1,20 +1,25 @@
 # Code Editor Shortcuts
 
-This [Obsidian](https://obsidian.md) plugin adds keyboard shortcuts (hotkeys) commonly found in code editors such as Visual Studio Code or Sublime Text.
+This [Obsidian](https://obsidian.md) plugin adds keyboard shortcuts (hotkeys) commonly found in code editors such as Visual Studio Code, Sublime Text, Emacs or Vim.
```

Comment:
I'll change this back for now because while I'm happy to have these few shortcuts added, I'm not prepared to support an additional two editors' worth of shortcuts (I have my hands full with existing feature requests as it is). And Obsidian already has a built-in option to enable Vim key bindings anyway.